contracts/interfaces/ICurrencyAmountConverter.sol

```diff
@@ -0,0 +1,14 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface ICurrencyAmountConverter {
```

Comment:
I thought "currency converter" may sound like it would turn A to B, but I suppose that would be "currency exchange". So this contract should probably be called CurrencyConverter.

Reply:
Ended up doing this in https://github.com/api3dao/claims-manager/pull/74/commits/aab0055fbab4a1124ff43a5e9abed3c584bcfe00